src/app/containers/PageHandlers/withPageWrapper.test.jsx

```diff
@@ -7,11 +7,11 @@ const dataProps = {
   service: 'news',
 };
 
-describe('with pageWarpper', () => {
+describe('with pageWrapper', () => {
   const PageWrapperContainer = () => <h1>Holla</h1>;
   const PageWrapperHOC = WithPageWrapper(PageWrapperContainer);
   shouldMatchSnapshot(
     `should render correctly`,
-    <PageWrapperHOC {...dataProps} />,
+    <PageWrapperHOC data={dataProps} service="news" />,
```

Comment:
why this change? I dont think service is an expected prop https://github.com/bbc/simorgh/blob/1805-update-brand-component-to-latest-version/src/app/containers/PageHandlers/withPageWrapper.jsx#L14

Reply:
Added because `PageWrapperHOC` uses the `default` service, and not the news service specified in dataProps - unless the service is passed to it as a prop

Reply:
This change isn't needed. Reverted it.